src/TestSuite/TestCase.php

```diff
@@ -161,6 +161,7 @@ public function tearDown()
             Configure::write($this->_configure);
         }
         $this->getTableLocator()->clear();
+        unset($this->_configure, $this->_pathRestore, $this->_tableLocator);
```

Comment:
Unsetting declared properties will make static analyzers unhappy. Set them to null / initial declared value instead.

Reply:
Setting to `null` or an empty array does not free all the memory though.